src/modules/git_commit.rs

```diff
@@ -0,0 +1,52 @@
+use super::{Context, Module, RootModuleConfig};
+use git2::Repository;
+
+use crate::configs::git_commit::GitCommitConfig;
+
+pub fn id_to_hex_abbrev(bytes: &[u8], len: usize) -> String {
```

Comment:
To maintain consistency with the other modules in the project, could we please move utility functions to the bottom of the file?